src/index.ts

```diff
@@ -103,10 +98,28 @@ export function getFileName(inPath: string) {
 }
 
 async function main() {
-    const renderer = new ImageRenderer(args.width, args.height, args.format, args.plddt, new FocusFirstResidue());
+    // Support multiple, comma-separated files
+    var files:string[] = args.in.split(',');
+
+    // Confirm all files exist
+    files.forEach(file => {
+        if (!fs.existsSync(file)) {
+            console.error(`Input path "${file}" does not exist`);
+            process.exit(1);
+        }
+    });
 
-    const fileName = getFileName(args.in);
-    const cif = await readCifFile(args.in);
+    // Run the rendering process
+    files.forEach(async file => {
+        await renderFile(file);
+    });
+}
+
+async function renderFile(filePath: string) {
```

Comment:
```suggestion
async function renderFile(filePath: string, renderer: ImageRenderer) {
```

and declare `const renderer = new ImageRenderer(args.width, args.height, args.format, args.plddt, new FocusFirstResidue());` in the `main` function. This will reuse the renderer for all the renders and should work out of the box sinde the render function always calls `this.canvas3d.clear();` at the end.